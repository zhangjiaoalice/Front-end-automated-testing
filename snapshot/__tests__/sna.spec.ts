import { config } from '../index.config';

describe('快照测试:', () => {
    // test('生成快照', () => {
    //     expect(config).toMatchSnapshot({
    //         date: expect.any(Date)
    //     });
    // });
    test('行内的快照测试', () => {
        expect(config).toMatchInlineSnapshot({
  date: expect.any(Date) }, `
Object {
  "baseUrl": "./",
  "date": Any<Date>,
  "dir": "/dist",
  "filename": "testFile",
}
`);
    });
})