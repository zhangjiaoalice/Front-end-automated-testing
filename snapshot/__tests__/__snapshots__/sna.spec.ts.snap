// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`快照测试: 生成快照 1`] = `
Object {
  "baseUrl": "./",
  "date": Any<Date>,
  "dir": "/dist",
  "filename": "testFile",
}
`;
